app-mobilephone/wammu/wammu-0.44.ebuild

```diff
@@ -1,27 +1,29 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-
+EAPI=6
 PYTHON_COMPAT=( python2_7 )
 
 inherit distutils-r1
 
 DESCRIPTION="Front-end for gammu to access mobile phones easily"
-HOMEPAGE="http://www.wammu.eu/"
-SRC_URI="http://dl.cihar.com/wammu/v0/${P}.tar.bz2"
+HOMEPAGE="https://www.wammu.eu/"
+SRC_URI="http://dl.cihar.com/wammu/v0/${P}.tar.xz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
 IUSE="bluetooth gnome"
 
-RDEPEND="|| ( ( >=app-mobilephone/gammu-1.36 dev-python/python-gammu )
-		>=app-mobilephone/gammu-1.25.0[python] )
+RDEPEND="
+	>=app-mobilephone/gammu-1.36
+	dev-python/python-gammu
 	>=dev-python/wxpython-2.8:*[${PYTHON_USEDEP}]
-	bluetooth? ( dev-python/pybluez[${PYTHON_USEDEP}]
+	bluetooth? (
+		dev-python/pybluez[${PYTHON_USEDEP}]
 		gnome? ( net-wireless/gnome-bluetooth )
-	)"
+	)
+"
 DEPEND="${RDEPEND}"
 
 # Supported languages and translated documentation
```

Comment:
this ebuild parses `LINGUAS`, which is bad. If the translations are installed into standard `/usr/share/locale`, then don't do this filtering and just rely on `INSTALL_MASK`

Reply:
Also, export `SKIPWXCHECK=yes` globally and don't override `distutils-r1` phase functions